.github/workflows/mysql-migtests.yml

```diff
@@ -107,3 +107,10 @@ jobs:
 
     - name: "TEST: mysql-reserved-words"
       run: migtests/scripts/run-test.sh mysql/reserved-words
+
+    - name: Create the live migration user
```

Comment:
Can we move this to live-grants.sql for mysql/pg respectively? 
If this is temporary, then pls add a TODO so that we remember to clean it up later.

Same for the PG workflow step.